src/Algebra/Operations/CommutativeMonoid.agda

```diff
@@ -6,7 +6,7 @@
 
 {-# OPTIONS --cubical-compatible --safe #-}
 
-open import Algebra
+open import Algebra using (CommutativeMonoid)
```

Comment:
```suggestion
open import Algebra.Bundles using (CommutativeMonoid)
```

Reply:
Ugh. I'm still making review suggestions on deprecated modules. 

Suggest that *no* such modules be updated as part of this, and all other such, PRs.